README.md

```diff
@@ -257,6 +257,23 @@ private URI create(URI url, T body) {
 }
 ```
 
+#### Create resource and retrieve its current state
+
+```java
+private URI create(URI url, String body) {
+    return unit.execute(POST, url, body).dispatch(series(),
+            on(SUCCESSFUL).dispatch(sameHeader("Location"),
+                    on("Content-Location").capture(),
+                    any(String.class).capture(location().andThen(location ->
```

Comment:
For this usecase also would be nice to have attribute available in capture(...) to avoid `location().andThen(...`
Also it is imposible to use original response inside nested execute